config/config.go

```diff
@@ -12,7 +12,8 @@ var (
 )
 
 func init() {
-	DebugEnabled, _ = strconv.ParseBool(Env("DEBUG", "false"))
+	DebugEnabled, _ = strconv.ParseBool(Env("DEBUG",
+		Env("ACTIONS_RUNNER_DEBUG", "false")))
```

Comment:
This will enable debug logging when running in CI with the "enable debug logging" checkbox checked